tests/apitests/python/library/configurations.py

```diff
@@ -2,25 +2,77 @@
 
 import base
 import swagger_client
+from swagger_client.rest import ApiException
+
+def set_configurations(client, expect_status_code = 200, expect_response_body = None, **config):
+    conf = swagger_client.Configurations()
+    if "project_creation_restriction" in config:
+        conf.project_creation_restriction = config.get("project_creation_restriction")
+    if "token_expiration" in config:
+        conf.token_expiration = config.get("token_expiration")
+    HasError = False
```

Comment:
Would `has_error` make more sense given the other variable names in this particular file (e.g., `token_expiration`)?

Reply:
Thanks, I renamed has_error to caught_api_exception, is it good?